code/modules/surgery/tools.dm

```diff
@@ -25,13 +25,22 @@
 	w_class = WEIGHT_CLASS_TINY
 	toolspeed = 0.5
 
-
 /obj/item/retractor/bone
 	name = "bone retractor"
 	desc = "Kinda looks like a chicken bone."
 	icon_state = "retractor_bone"
 	toolspeed = 1.25
 
+/obj/item/gastric_pump
+	name = "gastric pump"
+	desc = "a pump and a valve that connects directly to your stomach."
+	icon_state = "gastric_pump"
+	toolspeed = 0.85
+	materials = list(/datum/material/iron=3000, /datum/material/glass=1500)
+	flags_1 = CONDUCT_1
+	item_flags = SURGICAL_TOOL
+	tool_behaviour = TOOL_RETRACTOR
```

Comment:
TOOL_BONESET or a new define in code/_defines/tools.dm for like, TOOL_STOMPUMP or something, would be best here imo

Reply:
well rather if this is the basic one, a new define would probably be best

Reply:
yeah I forgot to do this oopsie :3